sdk/cognitivelanguage/Azure.AI.Language.Text/samples/Sample2_AnalyzeTextAsync_Sentiment.md

```diff
@@ -0,0 +1,192 @@
+    + "very uncomfortable. Once we notified the staff, they came and cleaned it and left candles.";
+
+string reviewC =
+    "Nice rooms! I had a great unobstructed view of the Microsoft campus but bathrooms were old and the"
+    + "toilet was dirty when we arrived. It was close to bus stops and groceries stores. If you want to"
+    + "be close to campus I will recommend it, otherwise, might be better to stay in a cleaner one.";
+
+// Prepare the input of the text analysis operation. You can add multiple documents to this list and
+// perform the same operation on all of them simultaneously.
+List<string> batchedDocuments = new()
+{
+    reviewA,
+    reviewB,
+    reviewC
+};
+
+AnalyzeTextInput body = new TextSentimentAnalysisInput()
+{
+    TextInput = new MultiLanguageTextInput()
+    {
+        Documents =
+        {
+            new MultiLanguageInput("A", reviewA) { Language = "en" },
+            new MultiLanguageInput("B", reviewB) { Language = "en" },
+            new MultiLanguageInput("C", reviewC) { Language = "en" },
+        }
+    },
+    ActionContent = new SentimentAnalysisActionContent()
+    {
+        OpinionMining = true,
+    }
+};
+
+Response<AnalyzeTextResult> response = await client.AnalyzeTextAsync(body);
+AnalyzeTextSentimentResult reviews = (AnalyzeTextSentimentResult)response.Value;
+
+Dictionary<string, int> complaints = GetComplaints(reviews);
+
+string negativeAspect = complaints.Aggregate((l, r) => l.Value > r.Value ? l : r).Key;
+Console.WriteLine($"Alert! major complaint is *{negativeAspect}*");
+Console.WriteLine();
+Console.WriteLine("---All complaints:");
+foreach (KeyValuePair<string, int> complaint in complaints)
+{
+    Console.WriteLine($"   {complaint.Key}, {complaint.Value}");
+}
+```
+
+Output:
+
+```text
+Alert! major complaint is *toilet*
+
+---All complaints:
+   food, 1
+   service, 1
+   toilet, 3
+```
+
+### Define method `GetComplaints`
+
+Implementation for calculating complaints:
+
+```C# Snippet:Sample2_AnalyzeTextAsync_Sentiment_GetComplaints
+private Dictionary<string, int> GetComplaints(AnalyzeTextSentimentResult reviews)
+{
+    Dictionary<string, int> complaints = new();
+    foreach (SentimentDocumentResultWithDetectedLanguage sentimentResponseWithDocumentDetectedLanguage in reviews.Results.Documents)
+    {
+        foreach (SentenceSentiment sentence in sentimentResponseWithDocumentDetectedLanguage.Sentences)
+        {
+            foreach (SentenceTarget target in sentence.Targets)
+            {
+                if (target.Sentiment == TokenSentiment.Negative)
+                {
+                    complaints.TryGetValue(target.Text, out int value);
+                    complaints[target.Text] = value + 1;
+                }
+            }
+        }
+    }
+    return complaints;
+}
+```
+
+See the [README] of the Text Analytics client library for more information, including useful links and instructions.
```

Comment:
Missed link again? Seems similar in other samples as well. Maybe I misunderstand but assuming these should have links

Reply:
I'm sorry I forgot this - in md : will [README] get autolinked?

Reply:
Link is referenced at the bottom of the file